app/javascript/angular/code/services/google/_map.js

```diff
@@ -71,17 +84,24 @@ export const map = (
       googleMaps.listenPanOrZoom(this.mapObj, callback);
     },
 
-    goToAddress: function(address) {
-      if (!address) return;
-
-      const callback = (results, status) => {
-        if (!googleMaps.wasGeocodingSuccessful(status)) return;
-
-        const latLngBounds = results[0].geometry.viewport;
-        this._fitBoundsWithoutPanOrZoomCallback(latLngBounds);
-      };
+    _goToAddress: function(place) {
+      if (!place.geometry) {
```

Comment:
Not sure how this works, but when I type something invalid and press enter I see an error in the console.

Reply:
That is what should happen when you type sth invalid :)

Reply:
I mean, I left the error to indicate that. It doesn't need to be written out but thought it might be helpful (but I guess it wasn't?).

Reply:
No, I just have this instinct that warnings are okish, but an error indicates something that should never happen just happend. Not a big deal though.

Reply:
hmmm ah, ok, I know what you mean. I'll check that

Reply:
<img width="506" alt="Screenshot 2019-08-26 at 18 09 51" src="https://user-images.githubusercontent.com/457999/63705092-c0300700-c82c-11e9-91a4-e814022ebc8b.png">

so you're seeing an error besides the above text?

Reply:
aaaaah, the error (`Uncaught Error: Trying to send an unexpected type of value through port locationUpdated:) is showing in Chrome only...

Reply:
we should take care of this, I'll add it to the board for now cos I need to finish other things

Reply:
Yes, I see this error. 
Sound like a good plan.